### ISSUE TITLE
do evaluation on OTB and VOT

### ISSUE THREAD

**Issue by zhangyujia0223** [HyeonseobNam/py-MDNet#5]
Could you tell me how to do evaluation on OTB and VOT based on python? 
Or could you publish the evaluation file?
I am learning tracking based on your architecture. 
Thank you very much!

**Comment by handworker** [comment]
@zhangyujia0223 , hello, do you evaluate the model which is provided by the author on the OTB-100 dataset? I test it and find that it is lower than the matlab version.

**Comment by zhangyujia0223** [comment]
@handworker I don't know how to do evaluation with python. I am beginner. Could you give me some examples? Thank you very much! I want to discuss with you.

**Comment by handworker** [comment]
@zhangyujia0223  Maybe you can write a script to test it on the OTB dataset (e.g., python run_tracker -s xx -d -f ), you only need to obtain all the sequence names and specify the name xx in the above.

**Comment by zhangyujia0223** [comment]
@handworker yeah, I have done that. But I want to do the evaluation on OTB and VOT (one pass evaluation--OPE). Especially I want to draw the precision plot and success plot. I think there are example codes to do that. But I didn't find it. Could you do that?

**Comment by handworker** [comment]
@zhangyujia0223 I only evaluate it on OTB with the matlab implementation. If you want to do that, firstly convert the results (i.e., json file) to the .mat file, and then you can follow the benchmark steps to obtain the OPE evaluation. Hope that it is helpful for you.

**Comment by zhangyujia0223** [comment]
@handworker Thanks very much. I'll try.

**Comment by zhangyujia0223** [comment]
@handworker I have gotten the .mat file of MDNet, how can I do OPE evaluation just on MDNet? Is there specifical function to do it? The benchmark is for too many trackers.
